Makefile

```diff
@@ -0,0 +1,36 @@
+default: 
+	@echo "Call a specific subcommand:"
+	@echo
+	@$(MAKE) -pRrq -f $(lastword $(MAKEFILE_LIST)) : 2>/dev/null\
+        | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}'\
+        | sort\
+        | egrep -v -e '^[^[:alnum:]]' -e '^$@$$'
+	@echo
+	@exit 1
+
+.state/docker-build-web: Dockerfile dev-requirements.txt base-requirements.txt
+	#Build web container for this project 
+	docker-compose build --build-arg IPYTHON=$(IPYTHON) --force-rm web
```

Comment:
I think the `--build-arg IPYTHON=$(IPYTHON)` here is extraneous as we don't have any build args for our Dockerfile.